src/installationFlag.ts

```diff
@@ -0,0 +1,17 @@
+import { ExtensionContext } from "./preRelease/types";
+
+const IS_JUST_INSTALLED_FLAG = "is-just-installed-tabnine";
```

Comment:
flag name is confusing - you set this key to `true` once you already handled the "just installed" logic, so it represent the opposite of `is-just-installed-tabnine` - its more like `already-installed` or something similar